website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +64,41 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px is considered disruptive to the user experience and may cover important information on the page.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. If a warning or error is contextual to a specific UI feature, such as a form, directly associate an Alert to that feature instead of a Toast.
 
-![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
+![](/assets/components/toast/toast-placement.png =600x*)
 
 ### Displaying multiple Toasts
 
-When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast.
+When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast. For consistency, each stacked Toast should be the same width.
+
+![](/assets/components/toast/toast-placement-multiple.png =600x*)
+
+## When to persist or timeout
+
+A Toast can either persist until the user dismisses it or be automatically dismissed via a timeout. 
+
+If a Toast contains actions or critical information (often associated with a warning or critical toast), we recommend that it persist until the user dismisses it. Keep in mind that when a Toast persists, it will remain visible across pages until dismissed.
+
+![Showing a critical Toast with a cluster failure message with a button that says retry and a link that says view cluster](/assets/components/toast/toast-persist-critical-retry.png)
+
+### Timeout best practices
+
+When a Toast is set to timeout, it should timeout after seven seconds, and the content should be no longer than a few of words, e.g., “Cluster created.” This is because when a Toast appears, there must be enough time for a user to notice it, then read it in time before it auto dismisses. 
```

Comment:
```suggestion
When a Toast is set to timeout, it should timeout after seven seconds, and the content should be no longer than a few words, e.g., “Cluster created.” This is because when a Toast appears, there must be enough time for a user to notice it, then read it in time before it auto dismisses. 
```